cpp/cppsdk/gsdkLinuxPch.h

```diff
@@ -3,5 +3,5 @@
 
 #pragma once
 
-#include "jsoncpp/json/json.h"
+#include "json/json.h"
```

Comment:
thanks for the PR! Does this work in Windows as well?

Reply:
Yes it's a linux only precompiled header include. You can see that `gsdkWindowsPch.h` has the same path. But perhaps I'm missing something? `jsoncpp/` folder does not exist anywhere.